### PR TITLE
Dev3

### PR DESCRIPTION
**Pull request by kofimeighan**

### CODE REVIEW

File: portfolio/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
-By default it contains a barebones web app. To run a local server, execute this
+by default it contains a barebones web app. To run a local server, execute this
 command:
 
 ```bash
 mvn package appengine:run
-```
\ No newline at end of file
+``
```
Comment (wangtong-g) [inline portfolio/README.md:8]: In markdown, this should be three `

File: portfolio/README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 This directory is where you'll write all of your code!
 
-By default it contains a barebones web app. To run a local server, execute this
+by default it contains a barebones web app. To run a local server, execute this
```
Comment (wangtong-g) [inline portfolio/README.md:3]: I think it should be uppercase.

File: portfolio/src/main/webapp/education.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    </ul>
+
+    <ul class="nav">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="education.html">Education & Activities</a></li>
+      <li><a href="pictures.html">Film Pictures</a></li>
+      <li><a href="mixes.html">Music Mixes</a></li>
+    </ul>
+
+
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:27]: Nitpicking: remove this extra line.

File: portfolio/src/main/webapp/education.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                 width=150px height=175px alt="Pine Crest Logo"></span>
+        <p><i><b>Pine Crest School</b></i> - Pine Crest School is a
+          private prepatory school in Fort Lauderdale, Florida.
+           I attended from August 2015 to May 2019.</p>
+
+      <span><img src="images/Columbia_Engineering_logo.png" 
+                 width=150px height=175px alt="Columbia University Crown">
+        <p><i><b>Columbia University</b></i> - I currently attend Columbia 
+          University in the City of New York with an anticipated major in 
+          Computer Science.My expected graduation date is May 2023.</p>
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:39]: Nitpicking: space between "My".

File: portfolio/src/main/webapp/education.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+              resources and a safe place to express myself.</p></br>
+
+        <span><img src="images/readahead.jpg" 
+                   height=150px width=350px alt="NSBE Logo"></span>
+        <p><i><b>Read Ahead</b></i> - I am a mentor with Read Ahead, 
+          an organization that connects adults and college students with
+            underpriviledged elementary and middle school kids in Morningside
+              Heights.</p>
+
+
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:66]: Nit: remove this extra line.

File: portfolio/src/main/webapp/education.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        To download my full resume, please click the icon
+        <a href="images/resume.pdf" download> 
+          <img src="images/download icon.png" alt="Resume Download"
+               width=25px height=25px></a>
+
+ </body>
+</html>
+
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:74]: Nit: remove these two lines after "</html>"

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -1,55 +1,44 @@
...
+      <p> My name is Kofi Meighan, and I am from Boynton Beach, Florida. I 
+        began my ventures within Computer Science when I was very young and 
+        I didn't even know it. I saw coding in action when I would program 
+        lego robots to move a designated period of time, a set number of 
+        rotations, or a very precisely calculated number of radians. 
+        That was the start, and finally being able to apply my skills to 
+        projects that will advance technology and positively develop 
+        someone's experience.
+
+        <img src="images/kofianddylan.jpg"
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/index.html:39]: There are three line wrapping methods in the style guide.  (https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping). 

Try not to mix them.

File: portfolio/src/main/webapp/education.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+        <p><i><b>Columbia University</b></i> - I currently attend Columbia 
+          University in the City of New York with an anticipated major in 
+          Computer Science.My expected graduation date is May 2023.</p>
+
+    <h2><u>Activities</u></h2>
+      <h5>At Columbia I partake in a few clubs and organizations,
+          I have listed most of them below:</h5>
+        <span><img src="images/Bacch B.png" 
+                   width=150px height=175px alt="Bacchanal Logo"></span>
+          <p><i><b>Bacchanal</b></i> - I am a member of Columbia University's
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/education.html:46]: There are three line wrapping methods in the style guide. (https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping).

Try not to mix them. Please fix all "<img" line wrapping in this file.

File: portfolio/README.md (hunk 1 above)
Comment (kofimeighan) [reply portfolio/README.md:8]: Added in the next PR.

File: portfolio/README.md (hunk 2 above)
Comment (kofimeighan) [reply portfolio/README.md:3]: Added in the next PR.

File: portfolio/src/main/webapp/education.html (hunk 3 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/education.html:27]: Removed.

File: portfolio/src/main/webapp/education.html (hunk 4 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/education.html:39]: Added.

File: portfolio/src/main/webapp/education.html (hunk 5 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/education.html:66]: Removed.

File: portfolio/src/main/webapp/education.html (hunk 6 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/education.html:74]: Removed !

File: portfolio/src/main/webapp/index.html (hunk 7 above)
Comment (kofimeighan) [reply portfolio/src/main/webapp/index.html:39]: I went through and applied a consistent wrapping method in all the files.

**Pull request merged**